src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -510,10 +533,19 @@ export const ArtworkSidebarCommercialFragmentContainer = createFragmentContainer
   {
     artwork: graphql`
       fragment ArtworkSidebarCommercial_artwork on Artwork {
-        slug
+        edition_sets: editionSets {
+          internalID
+          id
+          is_acquireable: isAcquireable
+          is_offerable: isOfferable
+          sale_message: saleMessage
+          ...ArtworkSidebarSizeInfo_piece
+        }
         internalID
-        is_for_sale: isForSale
+        isOfferableFromInquiry
+        isPriceHidden
         is_acquireable: isAcquireable
+        is_for_sale: isForSale
         is_inquireable: isInquireable
         is_offerable: isOfferable
```

Comment:
Sorry, I wasn't clear on my previous comment 😓 
We can remove these aliases and just have:
```suggestion
        isAcquireable
        is_for_sale: isForSale
        isInquireable
        isOfferable
```
I think we were using the aliases because of some outdated MP issue, so they're no longer needed.

Reply:
Got it. In that case I'm going to merge this and move that to another ticket. 1) Its outside of the scope 2) its not a blocker to this work 3) I would have to look into how this component's children consume its props to ensure no regressions are introduced and that too is outside of the scope of this work.